src/config/index.js

```diff
@@ -9,6 +9,10 @@ const config = {
   HASH_SALT: process.env.HASH_SALT,
   PASSWORD: process.env.PASSWORD,
   defaultCacheTtl: parseInt(process.env.DEFAULT_CACHE_TTL, 10),
+  sendgridKey: process.env.SENDGRID_KEY,
```

Comment:
Please update the docs with the process of setting up twilio sendgrid as well as how to set it up.